unit-tests/shared/src/test/require-jdk11/org/scalanative/testsuite/javalib/nio/file/FilesTestOnJDK11.scala

```diff
@@ -205,7 +205,7 @@ class FilesTestOnJDK11 {
     val ioPath = getCleanIoPath("StringBuilderSmall")
     val dataOut = getDataOut()
 
-    val output = jl.StringBuilder(dataOut)
```

Comment:
How did this work at all? Is there a problem the Java API?

Reply:
`AbstractStringBuilder` has this but no args.
```scala
object AbstractStringBuilder {
  final val INITIAL_CAPACITY = 16
}
```
Can't imagine that it would compile.